### PR TITLE
Update sejda-pdf-desktop to 2.5.0

### PR DESCRIPTION
**Pull request by gobinathm**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s an unrelated commit.

**Comment by gobinathm** [comment]
@vitorgalvao i have corrected it

File: Casks/sejda-pdf-desktop.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  sha256 'ec2dc5dcfea0dad3ef620ebdbd6d54b314b186cecd8ee7612431f14d18f06c83'
 
   # bitbucket.org/sejdapdf/sejda-desktop-public was verified as official when first introduced to the cask
   url "https://bitbucket.org/sejdapdf/sejda-desktop-public/downloads/sejda-desktop_#{version}.dmg"
   name 'Sejda PDF Desktop'
   homepage 'https://www.sejda.com/desktop'
 
   app 'Sejda PDF Desktop.app'
-end
+end
```
Comment (vitorgalvao) [inline Casks/sejda-pdf-desktop.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

Comment (gobinathm) [inline Casks/sejda-pdf-desktop.rb]: @vitorgalvao thanks for the suggestion. I missed it earlier.

**Pull request merged**